### ISSUE TITLE
Menu builder wrong items being deleted

### ISSUE THREAD

**Issue by iSWORD** [thedevdojo/voyager#2503]
- Laravel Version: 5.5.28
- Voyager Version: 1.0.13
- PHP Version: 7.0.22-0ubuntu0.16.04.1
- Database Driver & Version: NA

### Description:
Deleting items in the menu builder doesn't work properly if a delete operation was canceled by the user.

### Steps To Reproduce:
- Go to the menu builder
- Hit delete on one of the menu items (ITEM_A)
- Cancel the delete modal by clicking outside of it or clicking cancel
- Hit delete on another item (ITEM_B)
- Confirm your action on ITEM_B
- The first item (ITEM_A) will be deleted instead

### Cause:

https://github.com/the-control-group/voyager/blob/97df19285b2b3c8f50bc3a3840702295477e366a/resources/views/menus/builder.blade.php#L274

Once the __id is replaced by an actual id (number), the replacement will no longer work.

### Possible fix:

Change the way the id is injected into the action.

### Finally:

Thank you very much for all the great work you've done with Voyager.

P.S. I _may_ submit a PR to solve this if I had enough time over the weekend.

**Comment by iSWORD** [comment]
I solved it on my installation using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)

```javascript
/**
 * Delete menu item
 */
$('.item_actions').on('click', '.delete', function (e) {
    id = $(e.currentTarget).data('id');
    url = new URL($('#delete_form')[0].action);
    params = new URLSearchParams(url.search);
    params.set('id', id);
    url.search = params.toString();
    $('#delete_form')[0].action = url.toString();
    $('#delete_modal').modal('show');
});
```

Can't submit this as a PR since this doesn't work on IE or Edge [yet](https://caniuse.com/#search=URLSearchParams).

We could use other solutions like [this one](https://stackoverflow.com/a/6021027/1473350) though. This should be cross-browser.

**Comment by iSWORD** [comment]
Actually, this seems to be already fixed here;
https://github.com/the-control-group/voyager/blob/97df19285b2b3c8f50bc3a3840702295477e366a/resources/views/bread/browse.blade.php#L280

but the fix still needs to be ported to the menu builder.

**Comment by emptynick** [comment]
Fixed in #3953
